src/routes/Settings.tsx

```diff
@@ -11,124 +11,182 @@ import CustomizedInputBase from "./SmartTextBox";
-                    <ListItem>
-                        <ListItemText primary="Encryption Algorithm" />
-                        {/*<Divider sx={{ height: 28, m: 0.5 }} orientation="vertical" />*/}
-                        <Select
-                            labelId="demo-simple-select-label"
-                            id="demo-simple-select"
-                            value="argon2"
-                            label="Age"
-                        />
-                    </ListItem>
-                </Card>
-
-                    <Typography variant={'h4'}>Pastebin API</Typography>
-                    <Card classes={{root: classes.card}}>
-                    <ListItem>
-                        <ListItemText primary="PasteBin API Key" secondary="23ourwfodifkhjklfquhdkajdh" />
-                        {/*<Divider sx={{ height: 28, m: 0.5 }} orientation="vertical" />*/}
-                        <IconButton color="primary" sx={{ p: '10px' }} aria-label="directions">
-                            <ChevronRight />
-                        </IconButton>
-                    </ListItem>
-                </Card>
-                </List>
-        </div>
-    )
-}
\ No newline at end of file
+    });
+  };
+
+
+  const getSetting = (key: string) =>{
+    getItem(key, (data) => {
+      const res = data[Storage.ENC_MODE] || []
+      console.log(res); // returns nothing
+      return res;
+    })
+    return "Not Set"
+  };
+
+function getDataWrapper(key: string):any {
+  getItem(key, (data) => {
+      const res = data[key] || []
+      alert(res) 
+      console.log("res", res); // returns something
+      return res
+  })
+}
+
+
+  return (
+    <div>
+      <Typography variant='h2' className={classes.pageHeading} >Settings</Typography>
+      <List className={classes.list}>
+        <Typography variant={'h4'}>Dark Mode</Typography>
+        <Card classes={{ root: classes.card }}>
+          <ListItem>
+            <ListItemText
+              primary="Dark Mode" />
+            {/*<Divider sx={{ height: 28, m: 0.5 }} orientation="vertical" />*/}
+            <Checkbox defaultChecked onClick={() => setItem(Storage.THEME, true)} />
+          </ListItem>
+        </Card>
+
+        <Card classes={{ root: classes.card }}>
+          <ListItem>
+            <ListItemText primary="Encryption Algorithm" />
+            {/*<Divider sx={{ height: 28, m: 0.5 }} orientation="vertical" />*/}
+            <Select
+              onClick={() => setItem(Storage.ENC_MODE, "AES-GCM")}
+              labelId="demo-simple-select-label"
+              id="demo-simple-select"
+              value="AES-GCM"
+              label="AES-GCM"
+            />
+          </ListItem>
+        </Card>
+
+        <Typography variant={'h4'}>Pastebin API</Typography>
+        <Card classes={{ root: classes.card }}>
+          <ListItem>
+            <ListItemText primary="PasteBin API Key" secondary={getDataWrapper(Storage.API_KEY)} />
```

Comment:
probably setting this wrong, but "Not Set" works when calling the getSetting func